tests/main/install-errors/task.yaml

```diff
@@ -1,5 +1,9 @@
 summary: Checks for cli errors installing snaps
 
+details: |
+    Checks error messages and exit codes returned by various failing snap
```

Comment:
```suggestion
    Check error messages and exit codes returned by various failing snap
```

Reply:
Thanks! I'll change the summary to match as well